perfdash/go.mod

```diff
@@ -16,11 +15,33 @@ require (
 	k8s.io/kubernetes v1.15.0
 )
 
+require (
```

Comment:
nit: why do we need separate require section for it?

Reply:
We don't, but `go mod tidy` chose to do so for some reason. I added the imports to an existing section and `go mod tidy` seems fine with that.